blueprints/blueprint-replacing-guacamole.md

```diff
@@ -34,19 +34,20 @@ While we've been contemplating alternatives for a few weeks or even months, the
 
 ![](images/guacvsxterm.png)
 
-The reason I think this goes beyond a "bare bones example" for us is that `webssh2` also includes an example client, which shows a single terminal in the browser using xterm.js. It's not really meant to do much beyond showing basic capabilities. However, since our xterm.js implementation will be contained fully in `antidote-web`, we won't actually need to use this. For us, `webssh2` can and will be deployed as a separate service within our cluster. `antidote-web` will transition to a pure HTML/CSS/JS application, requiring no servlets or other server-side logic whatsoever. For this architecture, the Node.js application within `webssh2` should suffice very well.
+The reason I think `webssh2` describes itself as a "bare bones example" is that it also includes an example client, which shows a single terminal in the browser using xterm.js. It's not really meant to do much beyond showing basic capabilities. However, since our xterm.js implementation will be contained fully in `antidote-web`, we won't actually need to use this. For our purposes, the server-side component of `webssh2` can be deployed on its own, as a standalone service within our cluster, for the sole purpose of translating websocket streams to SSH connections. As a result, `antidote-web` will transition to a pure HTML/CSS/JS application, requiring no servlets or other server-side logic whatsoever to be maintained as part of that project. This is a much simpler, and cleaner solution.
 
 This solution has a few key benefits for the project:
 
 - `xterm.js` is a much more widely used terminal-in-browser solution. It doesn't push an end-to-end solution, but rather allows us to build the experience we want from base components. This gives us more flexibility to customize to fit our needs.
 - The entire solution (both client and server) is written in Javascript, which is in contrast to the combination of C, Java, and Javascript that Guacamole uses. This means it's much easier for web-savvy community members to contribute, and lets us treat the solution less like a black-box.
 - A drastic reduction in footprint and increase in performance will make it *much* easier to preview lesson content in selfmedicate. We've determined that the existing solution is actually a significant impediment to building lesson content, and fixing this should result in a better experience there, greatly boosting curriculum contribution engagement.
+- A much lighter-weight transport (websockets) combined with much better support for modern web standards means the terminal will no longer be an impediment to strong mobile support. Given that this work is taking place within the context of a larger project to re-vamp `antidote-web` as a whole, I view this as the final step in moving towards officially supporting mobile with this project (finally!)
```

Comment:
Mobile support is huge, especially with the trend of tablets replacing Notebooks and Desktops.